docs/technical-setup/environment-setup/install-r-rstudio.md

```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
 1. Install the **GNU Fortran compiler**.
 Click the link on the [Tools - R for Mac OS](https://mac.r-project.org/tools/) page to download the `gfortran` installer package and follow all installation instructions.
 
-<!--
-### Install R on Windows
+### Install R on Windows with WSL2
 
+These instructions will install R on the WSL2 side of your computer.
 
-1. Navigate to the [Windows download page on the CRAN website](https://cran.r-project.org/bin/windows/base/), and follow instructions to download and install R.
+1. Copy and paste this code into the Ubuntu terminal to install R:
+
+    ```sh
+    # install a few helper packages needed to install R and certain R packages
+    sudo apt install --no-install-recommends software-properties-common dirmngr libssl-dev
+
+    # add the signing key (by Michael Rutter) for these repos
+    wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | sudo tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc
+
+    # add the R 4.0 repo from CRAN -- adjust 'focal' to 'groovy' or 'bionic' as needed
```

Comment:
I would not expect anyone to know what this means.
```suggestion
    # add the R 4.0 repo from CRAN 
```

you could als use `sudo add-apt-repository "deb https://cloud.r-project.org/bin/linux/ubuntu $(lsb_release -cs)-cran40/"` to automatically set the version.

Reply:
I had that originally but changed it to jammy at the last minute 😬